app/src/main/java/org/dhis2/data/forms/dataentry/fields/scan/ScanBindings.kt

```diff
@@ -0,0 +1,73 @@
+package org.dhis2.data.forms.dataentry.fields.scan
```

Comment:
I think you could easily move this class to form module, i think you only need to resolve closekyboard() method and org.dhis2.R dependencies.